changelog

```diff
@@ -1,4 +1,52 @@
-version 0.10.7
+Version 0.10.8
+  * Bug fixes:
+    * Content bugs:
+      * Typo fixes. (@eebop, @Hannah-E-M, @roadrunner56, @TheGiraffe3, @tibetiroka)
+      * Shortened Darkrest's description to better fit description box. (@roadrunner56)
+      * Replaced duplicated words with weights in phrases. (@tibetiroka)
+      * The Hai festival jobs now have deadlines. (@ziproot)
+      * Updated Korath World-ship names for Remnant jobs. (@ziproot)
+      * Corrected sound reference for '"Benga" Reverse Thruster'. (@Saugia)
+    * Engine bugs:
+      * Fixed a crash when sorting ships without a ship selected with clang. (@warp-core)
+      * Fixed a crash with non-ASCII characters passed to cctype methods with clang. (@warp-core)
+      * Fixed a crash when sorting the ship list with no ships. (@warp-core)
+      * Properly reset the player's flagship after death. (@tibetiroka)
+      * Unicode byte order marks at the beginning of data files will no longer be tokenized. (@warp-core)
+      * The flagship will no longer move through wormholes ahead of escorts when using fleet jumping. (@Koranir)
+      * The message for a player escort being destroyed will now appear earlier and not be duplicated. (@nickshanks)
+      * Fixed edge cases for the battery only flight check warning. (@nickshanks)
+      * Fixed a crash when drawing cloaked ship and flagship outlines. (@tibetiroka)
+  * Game content:
+    * Mission changes:
+      * The Pact Recon missions will now no longer offer immediately after each other. (@Anarchist2)
+      * Reduced the pay and availability of Swiftsong jobs. (@roadrunner56)
+      * The Umbral system is now "mark"ed by the "Pirate Duel" mission, instead of being a waypoint. (@ziproot)
+    * Balance:
+      * Reduced `relative heat damage` by a factor of 1.5, for consistency with the mass rebalance in v0.10.7. (@Quantumshark)
+      * Reduced the difficulty of "Wanderers: Mind 6". (@TheGiraffe3)
+        * Wanderer escorts: +1 Tempest.
```

Comment:
```suggestion
        * Wanderer escorts:
            * +1 Tempest.
```
I feel like it should be the same as the others.

Reply:
I didn't use a new line because it's not the same as the others; there's only one item here.

Reply:
I agree with lumbar here.